packages/asc-ui/src/utils/hooks/useFocusWithArrows.tsx

```diff
@@ -8,12 +8,16 @@ import { FOCUSABLE_ELEMENTS } from './constants'
  * @param ref Component ref
  * @param rotating Jump to first item from last or vice versa
  * @param directChildrenOnly Useful if you don't want to focus on other focussable elements inside the child components of the ref
+ * @param horizontally In case you need to navigate horizontally, using left / right arrow buttons
  */
 const useFocusWithArrows = (
   ref: React.RefObject<HTMLElement>,
   rotating = false,
   directChildrenOnly = false,
+  horizontally = false,
```

Comment:
Probably need to refactor this to accept an object with options, now we need to pass 4 arguments

Reply:
Yeah, agreed this is a tad confusing right now.